### ISSUE TITLE
Docs describing new version and features

### ISSUE THREAD

**Issue by joaomdmoura** [rails-api/active_model_serializers#1237]
### **tl;dr**
Update new docs and cover new topics: https://github.com/rails-api/active_model_serializers/tree/master/docs

---
I've being personally pushing a new docs forward: https://github.com/rails-api/active_model_serializers/tree/master/docs
We have being updating it accordingly but there is still a long way to go.

We've an ambitious goal with this doc, cover:
- Usual doubts (avoid unnecessary issue creation)
- Migration between versions (help developer to get up to date)
- How to topics (enabling ppl to quickly find solution and how to use features)

The idea is to update this docs to cover all this.

---
related to issues:
- #1243 

related to PRs:
- #1229

**Comment by bf4** [comment]
ref:

- https://github.com/rails-api/active_model_serializers/issues/1074
- https://github.com/rails-api/active_model_serializers/issues/1005

**Comment by remear** [comment]
I think this is done except for the upgrading docs

**Comment by bf4** [comment]
@remear Yeah, there's just no canonical issue for this. I mean this should be, but there's a bunch of others.

**Comment by beauby** [comment]
Closing in favor of #1694.
